collections/choosing-projects/index.md

```diff
@@ -7,7 +7,11 @@ items:
  - SerenityOS/serenity
  - komodorio/helm-dashboard
  - cloudquery/cloudquery
+ - EddieHubCommunity/LinkFree
```

Comment:
It looks like you're part of this org, but it seems more like an open source group of 2k+ people, so we'll let it pass.